bootstrap/pkg/apis/apps/kfdef/v1alpha1/application_types.go

```diff
@@ -318,6 +319,14 @@ const DefaultCacheDir = ".cache"
 // specifying a YAML config file.
 // configFile is the path to the YAML file containing the KfDef spec. Can be any URI supported by hashicorp
 // go-getter.
+func isValidUrl(toTest string) bool {
```

Comment:
The comments above are not for the new isValidUrl, but for LoadKFDefFromURI func?

Reply:
Thanks @jinchihe , moved comment to LoadKFDefFromURI func.